packages/config/config/devices/0x014a/tstat-tbz500.json

```diff
@@ -0,0 +1,625 @@
+			"label": "2nd Stage Heat Enable",
+			"$import": "~/templates/master_template.json#base_enable_disable"
+		},
+		{
+			"#": "5",
+			"label": "Aux Heat Enable",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"defaultValue": 1
+		},
+		{
+			"#": "6",
+			"label": "2nd Stage Cool Enable",
+			"$import": "~/templates/master_template.json#base_enable_disable"
+		},
+		{
+			"#": "7",
+			"label": "Temperature Unit",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Celcius",
+					"value": 0
+				},
+				{
+					"label": "Fahrenheit",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "8",
+			"label": "Minimum Off Time",
+			"unit": "minutes",
+			"valueSize": 1,
+			"minValue": 5,
+			"maxValue": 9,
+			"defaultValue": 5
+		},
+		{
+			"#": "9",
+			"label": "Minimum Run Time",
+			"unit": "minutes",
+			"valueSize": 1,
+			"minValue": 3,
+			"maxValue": 9,
+			"defaultValue": 3
+		},
+		{
+			"#": "10",
+			"label": "Setpoint H/C Delta",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 3,
+			"maxValue": 15,
+			"defaultValue": 3
+		},
+		{
+			"#": "11",
+			"label": "H Delta Stage 1 ON",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 6,
+			"defaultValue": 1
+		},
+		{
+			"#": "12",
+			"label": "H Delta Stage 1 OFF",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 0
+		},
+		{
+			"#": "13",
+			"label": "H Delta Stage 2 ON",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 2,
+			"maxValue": 7,
+			"defaultValue": 2
+		},
+		{
+			"#": "14",
+			"label": "H Delta Stage 2 OFF",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 0
+		},
+		{
+			"#": "15",
+			"label": "H Delta Aux ON",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 3,
+			"maxValue": 8,
+			"defaultValue": 3
+		},
+		{
+			"#": "16",
+			"label": "H Delta Stage 3 OFF",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 7,
+			"defaultValue": 0
+		},
+		{
+			"#": "17",
+			"label": "C Delta Stage 1 ON",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 6,
+			"defaultValue": 1
+		},
+		{
+			"#": "18",
+			"label": "C Delta Stage 1 OFF",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 0
+		},
+		{
+			"#": "19",
+			"label": "C Delta Stage 2 ON",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 2,
+			"maxValue": 7,
+			"defaultValue": 2
+		},
+		{
+			"#": "20",
+			"label": "C Delta Stage 2 OFF",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 0
+		},
+		{
+			"#": "21",
+			"label": "Mechanical Status",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 256,
+			"unsigned": true,
+			"readOnly": true
+		},
+		{
+			"#": "22",
+			"label": "SCP Status",
```

Comment:
I don't think it is immediately clear what SCP is.

Reply:
Done